docs/src/index.rst

```diff
@@ -0,0 +1,23 @@
+*****************************************************************
+A Getting Started Guide to the Genomic Epidemiology of SARS-CoV-2
+*****************************************************************
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   :caption: Table of contents
+
+   self
```

Comment:
We should drop `self` from the table of contents to avoid confusion. When a user is on the index page, they are already at "self", but the current ToC renders a link to that page as the first item:

![image](https://user-images.githubusercontent.com/85372/130295476-acfd6864-db48-4c17-a76c-14126dac3de6.png)